frontend/pages/_app.js

```diff
@@ -1,7 +1,36 @@
+import React from "react";
+import { ApolloClient, InMemoryCache, ApolloProvider } from "@apollo/client";
+import { Global, css } from "@emotion/react";
+import getConfig from "next/config";
+import ToastProvider from "../providers/ToastProvider/ToastProvider";
+import { globalStyles } from "../theme";
 import "../styles/global.css";
 
 const App = ({ Component, pageProps }) => {
-  return <Component {...pageProps} />;
+  const {
+    publicRuntimeConfig: { apiUrl },
+  } = getConfig();
+
+  const client = new ApolloClient({
+    cache: new InMemoryCache(),
+    uri: apiUrl,
+  });
+  return (
+    <>
+      <Global
+        styles={css`
+          ${globalStyles}
+        `}
+      />
```

Comment:
Please elaborate on the approach used here

Reply:
Global is a component of emotion that allows one to add the same css to all pages. As such, I found it appropriate to add it to the _app.js where changes to it affect all pages.